dev/make-distribution.sh

```diff
@@ -42,6 +42,7 @@ function exit_with_usage {
   echo "usage:"
   cl_options="[--name] [--tgz] [--mvn <mvn-command>]"
   echo "make-distribution.sh $cl_options <maven build options>"
+  echo "Decrease build times with the -T option: -T 1C will result in Maven using one thread per core."
```

Comment:
It seems the `make-distribution.sh` with `-T` option doesn't work because the build fails at line [120](https://github.com/a-roberts/spark/blob/14811ad22b33787ecc0d8ef231b00ab1e8e59793/dev/make-distribution.sh#L120).